tools/ci/choose_workflows.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# -----------------------------------------------------------
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2019-2020 Datadog, Inc.
+# -----------------------------------------------------------
+
+import sys
+import argparse
+import traceback
+from typing import Optional
+from src.utils import export_env
+from src.ci_context import get_ci_context, CIContext
+
+
+def should_run_unit_tests(ctx: CIContext) -> bool:
+    print('⚙️ Resolving `unit tests` phase:')
+    return should_run_phase(
+        ctx=ctx,
+        trigger_env=context.trigger_env.DD_RUN_UNIT_TESTS,
+        build_env=context.build_env.DD_OVERRIDE_RUN_UNIT_TESTS,
+        pr_keyword='[x] Run unit tests',
+        pr_path_prefixes=[
+            'Sources/',
+            'Tests/DatadogTests',
+            'Tests/DatadogCrashReportingTests',
+            'Datadog/Datadog.xcodeproj/',
+        ],
+        pr_file_extensions=[]
+    )
```

Comment:
Other ideas I had for enabling a phases automatically:
- if the branch name matches some regexp (won't require passing ENVs manually for commits without PR),
- if there is a comment with given keyword in the PR (this way we don't need a checklist in the PR, but this will require restarting job manually after adding such comment),
- if a keyword appears in PR description, without the checklist (easy to break with a typo and miss necessary phases).